### ISSUE TITLE
LiteTheIronMode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [FreezeWarp/freeze-messenger#47]
```
See /v1 in historical source.

Need to port to new API.
```

Original issue reported on code.google.com by `reht...@gmail.com` on 22 Jun 2011 at 1:29
- Blocked on: #87

**Comment by FreezeWarp** [comment]
With the relative lightness of the new Bootstrap mode, going to mark this as wontfix.
